cypress/integration/onboarding/email/existing_email_adress_spec.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+function signup(email, name, pw) {
+    // 2. Go to /login
+    cy.visit('/login');
+
+    // 3. Click on sign up button
+    cy.get('#signup').click();
```

Comment:
Signup button is visible when `config.TeamSettings.EnableOpenServer` is set to true.  However, by [default it is set to false](https://github.com/mattermost/mattermost-server/blob/master/config/default.json#L87).  Could you add [API Cypress command](https://github.com/mattermost/mattermost-webapp/blob/master/cypress/support/api_commands.js) that updates system config settings?  Such will be used to set `EnableOpenServer` to true before testing and revert after testing this spec.

Reply:
Thanks for pointing that out! I've updated with the custom command to set Enable Open Server to true for the test and revert it false afterwards.